packages/broker/src/broker.ts

```diff
@@ -15,6 +15,7 @@ const logger = new Logger(module)
 
 export interface Broker {
     getNode: () => Promise<NetworkNodeStub>
+    getAddress: () => Promise<EthereumAddress>
```

Comment:
We can remove this function after the associated tests have been removed (first in main, and then merged back here)